doc/API.md

```diff
@@ -1,208 +1,209 @@
-## Elektra Initiative Overview
```

Comment:
Format still broken, you somehow change every line of this file. (Maybe different line endings?)

Reply:
The PR just [changes the line endings from `CRLF` to `LF`](https://github.com/ElektraInitiative/libelektra/pull/2800/commits/a49269638fa6c9f14412a3b2cbe0a55e2c3cc202), which should be fine in my opinion.

Reply:
Ok, we should unify the line endings, see #2801.